buildModules.js

```diff
@@ -0,0 +1,332 @@
+#! /usr/bin/env node
+
+/*
+Copyright The Infusion copyright holders
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-project/infusion/raw/main/AUTHORS.md.
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
+*/
+
+/* eslint-env node */
+"use strict";
+
+const build = {};
+const cpy = require("cpy");
+const dayjs = require("dayjs");
+const fg = require("fast-glob");
+const fs = require("fs");
+const mkdirp = require("mkdirp");
+const path = require("path");
+const {execSync} = require("child_process");
+const {minify} = require("terser");
+
+/**
+ * Command line arguments processed into an object.
```

Comment:
Would the use of a prepackaged system such as https://www.npmjs.com/package/minimist help here?

Reply:
Thanks for the tip on using minimist.